tasks/feature-flag.yml

```
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2023 Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: 2023 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
+- name: "Check if a feature flag is already enabled"
+  ansible.builtin.command:
+    cmd: "gitlab-rails runner 'is_feature_enabled = Feature.enabled?(:{{ gitlab_feature_flag.name }}); puts is_feature_enabled'"
+  register: "is_feature_enabled"
+  changed_when: false
+
+- name: "Enable or disable a feature flag"
```

I'd suggest to append the feature flag name to the task name to be able to identify which feature flag is handled:

```suggestion
- name: "Enable or disable feature flag {{ gitlab_feature_flag.name }}"
```